backend/controllers/prisonerProfile/prisonerQuickLook.ts

```diff
@@ -142,6 +142,23 @@ export default ({ prisonerProfileService, prisonApi, telemetry, offenderSearchAp
       return prisonerDetail?.indeterminateSentence ? 'Life sentence' : 'Not entered'
     }
 
+    const visitBalancesSection =
+      // @ts-expect-error ts-migrate(2339) FIXME: Property 'error' does not exist on type 'unknown'.
+      visitBalancesResponse.error || (visitBalances && Object.keys(visitBalances).length)
```

Comment:
so I guess we can't have remainingVo without Pvo being present, so just checking for no props works fine

Reply:
agreed